esy.json

```diff
@@ -2,19 +2,19 @@
   "name": "melange",
   "version": "7.2.0",
   "dependencies": {
-    "ocaml": "4.12.0+BS",
+    "ocaml": "4.12.0",
     "@opam/reason": "^3.7.0",
     "@opam/dune": "*",
     "@opam/dune-action-plugin": "*",
-    "@opam/ocamlfind": "1.8.1"
+    "@opam/melange-compiler-libs": "*"
   },
   "devDependencies": {
     "@opam/ocaml-lsp-server": "ocaml/ocaml-lsp:ocaml-lsp-server.opam",
     "web-tree-sitter": "0.18.0",
     "@opam/merlin": "*"
   },
   "resolutions": {
-    "ocaml": "melange-re/ocaml#75f22c872451c66a4c4aadc43abee55697268a57"
+    "@opam/melange-compiler-libs": "melange-re/melange-compiler-libs:melange-compiler-libs.opam#ac4946e775"
```

Comment:
It should not be a resolution then, this can be moved directly to `melange-compiler-libs` on the dependency list, also means users can remove all resolutions